### PR TITLE
Make "Deploy to OpenShift" command more discoverable

### LINKED ISSUE #406: Make the "Deploy to OpenShift" command more discoverable
Today, only people who know the "Deploy to OpenShift"  command exist will be able to use it. We need to figure out a way to make it more obvious

### PR DESCRIPTION
**Pull request by rgrunber**
- Fixes #406

Signed-off-by: Roland Grunberg <rgrunber@redhat.com>

deploy-quarkus-editor-titlebar-menu

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -213,6 +214,13 @@
           "command": "quarkusTools.deployToOpenShift",
           "when": "quarkusProjectExistsOrLightWeight"
         }
+      ],
+      "editor/title": [
+        {
+          "command": "quarkusTools.deployToOpenShift",
+          "when": "quarkusProjectExistsOrLightWeight",
+          "group": "navigation@1"
```
Comment (rgrunber) [inline package.json]: Not sure if this particular `group` is needed.

**Comment by rgrunber** [comment]
Should we also contribute the `Quarkus: Debug current Quarkus project` to the Run menu (beside the newly contributed Quarkus menu item) ?

~~Update : I also noticed we should probably restrict the deploy icon so it doesn't show up in any editor. Maybe only Java source files or application.properties files.~~

File: package.json (hunk 1 above)
Comment (rgrunber) [reply package.json]: According to https://code.visualstudio.com/api/references/contribution-points#Sorting-of-groups , we should probably make this `1_run@1`

Update : `1_run` doesn't seem to display it as an icon (but in the `...` context menu), so sticking with `navigation`.

**Comment by rgrunber** [comment]
quarkus-discoverable

**Comment by rgrunber** [comment]
CC'ing @mohitsuman in case he may have some suggestions as well (regarding icons/command exposure,etc).

**Comment by mohitsuman** [comment]
@rgrunber Can we use `OpenShift` icon in place of rocket icon. The icon can provide more intuitive way to identify it.

**Comment by rgrunber** [comment]
Yes, this is a fairly straightforward change. The main issue is that the icon doesn't quite blend in with the other icons at the smaller size.

openshift-icon-titlebar

We could try to import https://rawgit.com/openshift/openshift-logos-icon/master/demo.html (icon-openshift) and see if that looks better integrated.

**Comment by rgrunber** [comment]
openshift-icon-titlebar2

Thoughts on this vs. the one in previous comment ?

**Comment by mohitsuman** [comment]
I think the last one looks much better and it also goes in alignment with the extension icon. So the icon means we lead them to the extension with the `Deploy to OpenShift` command.

**Comment by rgrunber** [comment]
Merged as 2ca2a8179520ba701617d28293e2e2c0f836106e.

**Pull request merged**